### ISSUE TITLE
Ability to add missing Items into the list being displayed.

### ISSUE THREAD

**Issue by mahit777** [ng-select/ng-select#806]
It would be a very powerful feature to add missing Items to the list although it would be more complex when dealing with objects. It would be great if it is present in this component

**Comment by varnastadeus** [comment]
Can you rephrase your suggestion? What do you mean by missing items?

**Comment by mahit777** [comment]
@varnastadeus  thank you for replying.  What I meant was If the filter query is not present in the list of  Items the user can add custom values. 
Say I have dropdowns for a list of skillsets a person has and as that can be a very big set I would let the user enter if the list is missing any. I hope I made it clear :)

**Comment by varnastadeus** [comment]
@mahit777 it already implemented, use [addTag] also you could check demos
